audbackend/core/backend/base.py

```diff
@@ -71,6 +71,70 @@ def checksum(
             path,
         )
 
+    def _copy_file(
+            self,
+            src_path: str,
+            dst_path: str,
+            verbose: bool,
+    ):  # pragma: no Windows cover
```

Comment:
We need this because `FileSystem` and `Artifactory' provide a custom implementation and `SingleFolder`, which relies on the default implementation, is not tested on Windows.

Reply:
It's not ideal to skip testing the default implementation on Windows.
What was the reason that `SingleFolder` is not tested under Windows?